applications/ShapeOptimizationApplication/python_scripts/response_functions/plane_packaging_response.py

```diff
@@ -0,0 +1,63 @@
+# ==============================================================================
+#  KratosShapeOptimizationApplication
+#
+#  License:         BSD License
+#                   license: ShapeOptimizationApplication/license.txt
+#
+#  Main authors:    Geiser Armin, https://github.com/armingeiser
+#
+# ==============================================================================
+import KratosMultiphysics as KM
+
+from .packaging_response_base import PackagingResponseBase
+from .. import custom_math as cm
+
+
+class PlanePackagingResponse(PackagingResponseBase):
+    """
+    A class for plane packaging response function. The plane is defined by an origin point and a normal vector.
```

Comment:
Imho as a non-native speaker, the first sentence doesn't sound very English. 

```suggestion
    A class that defines the response function for plane-based packaging. The plane is defined by an origin point and a normal vector.
```
The change should be analogously done for the Mesh-based packaging.